Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -890,6 +890,16 @@ public static void RemovePlayerForConnection(NetworkConnection conn, bool destro
         // players on a single client
         static void OnCommandMessage(NetworkConnection conn, CommandMessage msg, int channelId)
         {
+            if (!conn.isReady)
```

Comment:
this looks like it should be a separate PR for easier bisecting later?

Reply:
Done